cmd/nodecmd/local.go

```diff
@@ -242,3 +263,265 @@ func notImplementedForLocal(what string) error {
 	ux.Logger.PrintToUser("Unsupported cmd: %s is not supported by local clusters", logging.LightBlue.Wrap(what))
 	return nil
 }
+
+func newLocalValidateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "validate [clusterName]",
+		Short: "Validate a specified L1 with an Avalanche Node set up on local machine (PoS only)",
+		Long: `Use Avalanche Node set up on local machine to set up specified L1 by providing the
+RPC URL of the L1. 
+
+This command can only be used to validate Proof of Stake L1.`,
+		Args: cobra.MaximumNArgs(1),
```

Comment:
I believe it has 1 arg, not max args 1

Reply:
or add more logic so, if there are not args, it asks for a new cluster name and starts it